### ISSUE TITLE
panic: unaligned 64-bit atomic operation - armv7l - v1.70

### ISSUE THREAD

**Issue by syepes** [VictoriaMetrics/VictoriaMetrics#1944]
**Describe the bug**
Just tried to upgrade my home raspberry pi 4 monitoring and it looks like the v1.70 version has some panic issue.
Rolling-back to the v1.68.0 works.

**Logs**
```
2021-12-14T12:40:22.537Z        info    VictoriaMetrics/lib/logger/flag.go:12   build version: victoria-metrics-20211202-133622-tags-v1.70.0-0-g7f2f26b25
2021-12-14T12:40:22.537Z        info    VictoriaMetrics/lib/logger/flag.go:13   command line flags
2021-12-14T12:40:22.537Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "httpListenAddr"=":8428"
2021-12-14T12:40:22.537Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "influxSkipSingleField"="true"
2021-12-14T12:40:22.537Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "loggerLevel"="INFO"
2021-12-14T12:40:22.538Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "memory.allowedPercent"="90"
2021-12-14T12:40:22.538Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "retentionPeriod"="2y"
2021-12-14T12:40:22.538Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "selfScrapeInterval"="30s"
2021-12-14T12:40:22.538Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "storageDataPath"="/storage"
2021-12-14T12:40:22.538Z        info    VictoriaMetrics/app/victoria-metrics/main.go:52 starting VictoriaMetrics at ":8428"...
2021-12-14T12:40:22.538Z        info    VictoriaMetrics/app/vmstorage/main.go:89        opening storage at "/storage" with -retentionPeriod=2y
2021-12-14T12:40:22.538Z        info    VictoriaMetrics/lib/memory/memory.go:37 limiting caches to 1932735282 bytes, leaving 214748365 bytes to the OS according to -memory.allowedPercent=90
2021-12-14T12:40:22.538Z        info    VictoriaMetrics/lib/storage/storage.go:970      loading MetricName->TSID cache from "/storage/cache/metricName_tsid"...
panic: unaligned 64-bit atomic operation

goroutine 1 [running]:
runtime/internal/atomic.panicUnaligned()
        runtime/internal/atomic/unaligned.go:8 +0x24
runtime/internal/atomic.Load64(0x20f80cc)
        runtime/internal/atomic/atomic_arm.s:286 +0x14
github.com/VictoriaMetrics/VictoriaMetrics/lib/workingsetcache.(*Cache).UpdateStats(0x20f80a0, 0x2253814)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/workingsetcache/cache.go:247 +0x104
github.com/VictoriaMetrics/VictoriaMetrics/lib/storage.(*Storage).mustLoadCache(0x213a500, {0x60d7b9, 0x10}, {0x60d20b, 0xf}, 0x2851eb84)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/storage/storage.go:974 +0x168
github.com/VictoriaMetrics/VictoriaMetrics/lib/storage.OpenStorage({0xbec64efe, 0x8}, 0xeaf625800, 0x0, 0x0)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/storage/storage.go:200 +0x68c
github.com/VictoriaMetrics/VictoriaMetrics/app/vmstorage.InitWithoutMetrics(0x7dc238)
        github.com/VictoriaMetrics/VictoriaMetrics/app/vmstorage/main.go:92 +0x41c
github.com/VictoriaMetrics/VictoriaMetrics/app/vmstorage.Init(0x7dc238)
        github.com/VictoriaMetrics/VictoriaMetrics/app/vmstorage/main.go:70 +0x1c
main.main()
        github.com/VictoriaMetrics/VictoriaMetrics/app/victoria-metrics/main.go:55 +0x24c
```

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
$ ./victoria-metrics-prod --version
version: victoria-metrics-20211202-133622-tags-v1.70.0-0-g7f2f26b25
```

**Comment by valyala** [comment]
The bug has been already fixed in the commit da402fbdfa53b51adf1281016573bd7b49af235c , which will be included in the next release. In the mean time it is possible to build VictoriaMetrics from this commit according to [these docs](https://docs.victoriametrics.com/#how-to-build-from-sources).

**Comment by syepes** [comment]
Thanks

**Comment by valyala** [comment]
FYI, the bugfix has been included in VictoriaMetrics [v1.71.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
